tests/unit/io/test_fileio.py

```diff
@@ -89,23 +86,7 @@ def test_find_files_when_file_absent(tmp_path):
     temp_file = os.path.join(tmp_path, TEMPORARY_FILE_NAME)
     with open(temp_file, "w"):
         with pytest.raises(StopIteration):
-            assert next(io_utils.find_files(str(tmp_path), "abc*.*"))
-
-
-@pytest.mark.parametrize("filesystem", REMOTE_FS_PREFIX)
-def test_is_remote_when_using_remote_prefix(filesystem):
-    """is_remote returns True when path starts with one of
-    the ZenML remote file prefixes"""
-    some_random_path = os.path.join(filesystem + "some_directory")
-    assert io_utils.is_remote(some_random_path)
-
-
-@given(text())
-def test_is_remote_when_using_non_remote_prefix(filesystem):
-    """is_remote returns False when path doesn't start with
-    a remote prefix"""
-    some_random_path = os.path.join(filesystem + "some_directory")
-    assert io_utils.is_remote(some_random_path) is False
```

Comment:
why were those tests removed?

Reply:
Mistake. Added back in now.

Reply:
Actually, if the tests were for something in fileio, then I moved the tests over to fileio tests instead of having them here.